pylot/perception/messages.py

```diff
@@ -59,6 +59,35 @@ def __str__(self):
             'height: {})'.format(self.timestamp, self.width, self.height)
 
 
+class DepthFrameMessage(Message):
```

Comment:
We already have a DepthFrameMessage class here: [https://github.com/erdos-project/pylot/blob/master/pylot/simulation/messages.py#L35](https://github.com/erdos-project/pylot/blob/master/pylot/simulation/messages.py#L35) . Can you please use it instead of introducing a new one?

Reply:
The DepthFrameMessage class in simulation requires a transform argument, and I'm not sure if I did it in the right way. Could you please check my new commit? Also, there's no runtime argument. Is it better to delete the runtime part from the DepthEstOperator or add runtime to DepthFrameMessage?

Reply:
re runtime: It is fine to not pass runtime to DepthFrameMessage. The runtime logic is a hack anyways because we don't currently have a way of automatically tracking the runtimes of callbacks.

re transform: I would pass the transform as an argument to the __init__ method of the Depth estimation operator so that the operator is agnostic to the particular camera setup its working with. If you do that, then you can pass in the center camera transform (i.e., bgr_camera_setup.transform) to the DepthEstimationOperator (here: https://github.com/erdos-project/pylot/blob/master/pylot/simulation/challenge/ERDOSAgent.py#L128).